R/learnerHelp.R

```diff
@@ -0,0 +1,214 @@
+#' @title Access help page of learner functions.
+#'
+#' @description Interactive function that gives the user quick access to the
+#'   help pages associated with various functions involved in the given learner.
+#' @template arg_learner
+#' @export
+#' @family learner
+#' @family help
+learnerHelp = function(learner) {
+  learner = checkLearner(learner)
+  callees = learner$callees
+  if (identical(callees, "")) {
+    callees = NULL
+  }
+  mlr.help = utils::help(learner$id, package = "mlr")
+  if (length(mlr.help) > 0) {
+    # we can give some more help
+    callees = c("mlr specifics", callees)
+  }
+  if (!is.null(callees)) {
+    n = 1
+    if (length(callees) > 1) {
+      repeat {
+        cat("Choose help page:\n")
+        cat(paste0(seq_along(callees), " : ", callees, "\n", collapse=""))
+        n <- readline("0 : cancel\n...: ")
+        n <- ifelse(grepl("\\D",n),-1,as.integer(n))
+        if (is.finite(n) && n >= 1 && n <= length(callees)) {
+          break
+        }
+	if (identical(n, 0L)) {
+          return(invisible(NULL))
+        }
+        catf("Invalid input. Enter a number between 1 and %d", length(callees))
+      }
+    }
+    if (length(mlr.help) > 0) {
+      if (n == 1) {
+        return(mlr.help)
+      }
+      n = n - 1
+    }
+    for (pkg_ref in stri_replace_all(learner$package, "", regex = "[+!_]")) {
+      h = utils::help(callees[n], package = (pkg_ref))
+      if (length(h) > 0) {
+        return(h)
+      }
+    }
+  }
+  catf("No information about learner %s found.",
+    coalesce(learner$name, learner$shortname, learner$id))
+  invisible(NULL)
+}
+
+#' @title Get specific help for a learner's parameters.
+#'
+#' @description Print the description of parameters of a given learner. The description
+#'   is automatically extracted from the help pages of the learner, so it may be incomplete.
+#'
+#' @template arg_learner
+#' @param param [\code{character}]\cr
+#'   Parameter(s) to describe. If no argument is given, information on the documentation status
+#'   of all parameters is printed.
+#' @export
+#' @family learner
+#' @family help
+learnerParamHelp = function(learner, param) {
+  learner = checkLearner(learner)
+  if (missing(param)) {
+    if (length(learner$help.list) > 0) {
+      cat("Documentation for the following parameters is present:\n")
+      for (p in names(learner$help.list)) {
+        catf("- %s (%s)", p,
+          stri_match_first_regex(learner$help.list[[p]], "(?<= )[^ ]*::[^\\n]*"))
+      }
+    }
+    missingdoc = setdiff(getParamIds(learner$par.set), names(learner$help.list))
+    if (length(missingdoc) > 0) {
+      catf("The following parameters are missing documentation:\n%s",
+        collapse(strwrap(collapse(missingdoc, sep = ", "), indent = 2), "\n"))
+    }
+    return(invisible(NULL))
+  }
+  assertCharacter(param, any.missing = FALSE)
+  all.param = getParamIds(learner$par.set)
```

Comment:
Maybe use `getParamIds(getLearnerParamSet(learner))`  here? 
Otherwise this might not work if `learner` is a wrapped learner?